src/pages/dashboard/messages.tsx

```diff
@@ -0,0 +1,52 @@
+import { hardcodedUsername, postWithAuth } from "@/auth";
```

Comment:
Can you add documentation both here, the component, and the README that this is a svix-react example?

To be honest, I'd probably not call the files Messages but rather SvixReactExample, or maybe put it in a subdirectory so it doesn't confuse people who don't want it / need it. Because it's more advanced.

Actually, maybe put it in an "advanced" directory or something?

I just really don't want to scare away people, they need to understand how simple Svix is. :)